src/FeedbackItem.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace Wheniwork\Feedback;
+
+use Spark\Data\EntityInterface;
+use Spark\Data\Traits\EntityTrait;
+
+class FeedbackItem
+{
+    use EntityTrait;
+
+    const POSITIVE = 'positive';
+    const PASSIVE = 'passive';
+    const NEGATIVE = 'negative';
+    const NEUTRAL = 'neutral';
+
+    private $body;
+    private $source;
+    private $link;
+    private $title;
+    private $rating;
+    private $sender;
+    private $tone;
+
+    private function types()
+    {
+        return [
+            'body' => 'string',
+            'source' => 'string',
+            'link' => 'string',
+            'title' => 'string',
+            'sender' => 'string',
+            'tone' => 'string'
```

Comment:
Normally everything will be a string. Nothing wrong with being explicit, I just tend to skip typing anything that is a string.